ext/node/polyfills/_fs/_fs_readFile.ts

```diff
@@ -64,7 +66,13 @@ export function readFile(
 
   const encoding = getEncoding(optOrCallback);
 
-  const p = Deno.readFile(path);
+  let p: Promise<Uint8Array>;
+  if (path instanceof FileHandle) {
+    const fsFile = new Deno.FsFile(path.fd);
+    p = Deno.readAll(fsFile);
```

Comment:
This is a deprecated function. Are there alternatives?

Reply:
I believe you can import `readAll` from `ext:deno_io/12_io.js.

https://github.com/denoland/deno/blob/main/ext/io/12_io.js#L117

Reply:
Thanks!

[5a61461](https://github.com/denoland/deno/pull/19294/commits/5a614610ca4a4ea1965be9d53c896315879b88f7) I use it